src/runtime/types/index.ts

```diff
@@ -65,10 +73,14 @@ export interface MessagesLoader {
 }
 
 export interface ConfigureOptions {
-  fallbackLocale: string;
-  formats?: Partial<Formats>;
-  initialLocale?: string;
-  loadingDelay?: number;
-  warnOnMissingMessages?: boolean;
-  ignoreTag?: boolean;
+  fallbackLocale: string | null | undefined;
```

Comment:
I'd say this is mandatory

Reply:
3.
Thank you, I made it optional because undefined is supplied in test. I've made it required and leave test as it is with `as any`.